subsys/fs/shell.c

```diff
@@ -520,6 +574,9 @@ SHELL_STATIC_SUBCMD_SET_CREATE(sub_fs,
 #endif
 	SHELL_CMD(pwd, NULL, "Print current working directory", cmd_pwd),
 	SHELL_CMD_ARG(read, NULL, "Read from file", cmd_read, 2, 255),
+	SHELL_CMD_ARG(cat, NULL,
+		"concatenate files and print on the standard output",
```

Comment:
Upper case first letter, as all the other commands do.

Reply:
Thanks! fixed.